source/howto/DockerHubMirror.rst

```diff
@@ -0,0 +1,148 @@
+.. _docker_mirror_guide:
```

Comment:
This document isnt included in any toctree, could you add a link to it on the `how to` toctree?

Reply:
:+1: Sphinx doesn't seem to be playing ball with this, hopefully my recently push works